config/install/block.block.az_barrio_main_menu.yml

```diff
@@ -4,7 +4,6 @@ dependencies:
   config:
     - system.menu.main
   module:
-    - block_class
```

Comment:
We should test these on blocks with and without classes added.

Reply:
Done ✅